### PR TITLE
August 2020 release notes

### PR DESCRIPTION
**Pull request by olenagerasimova**
I've tried to add blog post about key features implemented in August.

### CODE REVIEW

**Comment by olenagerasimova** [comment]
@g4s8 @olegmoz please review

File: _posts/2020/08/2020-08-13-release-notes.md [hunk 1]
```diff
@@ -0,0 +1,49 @@
...
+
+{% jb_picture_body %}
+
+[v0.10](https://github.com/artipie/artipie/releases/tag/0.10) was released on August 31, 
+here is the list of key features introduced this month:
+
+### Maven adapter
+Maven deploy workflow improved:
+- support checksum and etag headers
+- artifacts validation on upload
```
Comment (g4s8) [inline _posts/2020/08/2020-08-13-release-notes.md]: @olenagerasimova maybe we can add links to the tickets here?

**Review by g4s8 — CHANGES_REQUESTED** [review]
@olenagerasimova please check my comment

**Comment by yegor256** [comment]
@g4s8 @olegmoz looks good to me

**Comment by olenagerasimova** [comment]
@g4s8 thanks, I've added links

**Review by g4s8 — COMMENTED** [review]
@olenagerasimova thanks

**Review by g4s8 — APPROVED** [review]

**Comment by olegmoz** [comment]
@yegor256 it seems that https://blog.artipie.com/ does not show this post after merge, could you please look into it?

**Comment by yegor256** [comment]
@olenagerasimova @olegmoz it's published. See the changes I've made in 85985a9 We can't use `jekyll-bits` plugin, because GitHub publishes Jekyll sites in "safe" mode: https://jekyllrb.com/docs/github-pages/

**Comment by olegmoz** [comment]
@yegor256 thank you!

**Pull request merged**